e2e/cypress/integration/onboarding/use_team_invite_link_to_sign_up_spec.js

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @onboarding
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl, getRandomId} from '../../utils';
+
+describe('Onboarding', () => {
+    let testTeam;
+    const randomId = getRandomId();
+    const username = `user${randomId}`;
+    const email = `user${randomId}@sample.mattermost.com`;
+    const password = 'passwd';
+
+    const baseUrl = Cypress.config('baseUrl');
+    const mailUrl = getEmailUrl(baseUrl);
+
+    before(() => {
+        // # Do email test if setup properly
+        cy.apiEmailTest();
+
+        // # Update config to require email verification
+        cy.apiUpdateConfig({
+            EmailSettings: {
+                RequireEmailVerification: true,
```

Comment:
This will break succeeding tests since the email address of `sysadmin` test account we're using is not verified. With that, I suggests for you to add API command for [cy.apiVerifyUserEmailById](https://api.mattermost.com/#tag/users/paths/~1users~1{user_id}~1email~1verify~1member/post) and [use for sysadmin during global before hook](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/index.js#L98-L108), like
```javascript
        if (user.id) {
            // # Login existing sysadmin
            cy.apiAdminLogin().then(({user: sysadmin}) => {
                if (!sysadmin.email_verified) {
                    cy.apiVerifyUserEmailById(user.id)
                }
            });
        } else {
            // # Create and login a newly created user as sysadmin
            cy.apiCreateAdmin().then(({sysadmin}) => {
                cy.apiAdminLogin().then(() => {
                    cy.apiVerifyUserEmailById(sysadmin.id)
                    cy.apiSaveTutorialStep(sysadmin.id, '999');
                });
            });
        }
```